### PR TITLE
proto/profile.proto: fix typo

### PR DESCRIPTION
**Pull request by srenatus**

### CODE REVIEW

**Review by kalyanac — DISMISSED** [review]

File: proto/profile.proto [hunk 1]
```diff
@@ -109,7 +109,7 @@ message Sample {
   // entry in Profile.sample_type. All samples must have the same
   // number of values, the same as the length of Profile.sample_type.
   // When aggregating multiple samples into a single sample, the
-  // result has a list of values that is the elemntwise sum of the
+  // result has a list of values that is the elementwise sum of the
```
Comment (kalyanac) [inline proto/profile.proto:112]: Please make it "element-wise"

Comment (srenatus) [reply proto/profile.proto:112]: Sure!

Comment (srenatus) [reply proto/profile.proto]: ✔️ changed

**Comment by srenatus** [comment]
Ping! It's not like the world would stop without this fixed, but can we just merge it? 😃

**Comment by srenatus** [comment]
Thanks 🍰

**Comment by aalexand** [comment]
> Thanks 🍰

I actually haven't merged, I only rebased :) Some tests are failing for some reason.

**Comment by srenatus** [comment]
@aalexand yeah but you've reacted 😉 thanks for that.

**Comment by aalexand** [comment]
@srenatus CLA check fails now for some reason. Do you have the CLA signed?

**Comment by srenatus** [comment]
@aalexand I've switched jobs. I'll follow up on this issue! Sorry for delaying this.

**Comment by srenatus** [comment]
@aalexand Is there anything I can do to re-trigger a CLA check? I think it should be sorted now.

**Comment by kalyanac** [comment]
Restarted checks after #580 is merged.

**Comment by kalyanac** [comment]
@srenatus CLA is still not able to verify that you signed it. PR cannot be merged until it is signed.

**Comment by srenatus** [comment]
@kalyanac thanks, I'll try to take care of this.

**Comment by kalyanac** [comment]
@srenatus CLA bot is still not able to verify that you signed it. PR cannot be merged until it is signed.

**Comment by srenatus** [comment]
@kalyanac yeah it's on us. We're looking into it. Please bear with me 😄

**Comment by srenatus** [comment]
@googlebot I signed it!

**Comment by srenatus** [comment]
Fiiinaaaalllly 🎉

**Review by kalyanac — APPROVED** [review]

**Pull request merged**